src/test/java/ExamTest/SearchJobTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import pages.HomePage;
+import pages.JobPage;
+
+public class SearchJobTest {
+
+    WebDriver driver;
+    @BeforeMethod
+
+    public void BeforeClass(){
+        System.setProperty("webdriver.chrome.driver", "src\\main\\resources\\chromedriver.exe");
+        driver=new ChromeDriver();
+        driver.get("https://staff.am");
+    }
+
+    @Test
+    public void  HomePage() {
+
+        HomePage homePage = new HomePage(driver);
+        homePage.searchJob();
+
+        JobPage page=new JobPage(driver);
+        page.JobPage();
+
+    }
```

Comment:
This is incomprehensible test method, which doesn't tell reviewer or whoever reads the code, what it does actually.
Maybe you'll divide some of your page object methods smaller, so test method can be readable and test steps become obvious.